esphome/components/ethernet/__init__.py

```diff
@@ -145,6 +154,18 @@
     }
 )
 
+LLDP_SCHEMA = cv.Schema(
+    {
+        cv.Optional(CONF_ENABLED, default=False): cv.boolean,
```

Comment:
Is there a case when you would configure `lldp` in yaml but want it disabled? If so that is fine, but we can probably default it to True as we can assume you do want it enabled if you are configuring it.
```suggestion
        cv.Optional(CONF_ENABLED, default=True): cv.boolean,
```

Reply:
Not really, no - the specific reason for `ethernet.lldp.enabled` existing was so the LLDP transmitter could be enabled without modifying any of the default values, and without the ugliness of`lldp: {}` YAML.

I'd had a look around the code where there are examples handling both:

`feature: true`

and:

```
feature:
  key: val
  key2: val2
```

but it all looked rather ad-hoc, so I didn't want to go there without guidance.

Your suggestion makes `lldp: {}` work as would more reasonably be expected - LLDP on, with default settings - happy to make that change if a more optimal solution doesn't come to mind.

Reply:
Made the default=True change, then thought more about it and did away with `enabled` entirely.